### ISSUE TITLE
Add tags to resources card 

### ISSUE THREAD

**Issue by Gw3i** [OctobearHub/octobear.org#24]
It would be nice to see the tags for each resource card, so you understand which topic they belong to.
Maybe in the future you can click on the tag and just see all resources related to it. I think that would be a great additional feature.

The tags could be also displayed at the top, so you can filter upfront.

<img width="842" alt="Screenshot 2022-10-05 at 20 50 47" src="https://user-images.githubusercontent.com/101993818/194139018-65dbc696-351b-4c42-8fbb-2ab4f4fca4c2.png">
<img width="991" alt="Screenshot 2022-10-05 at 20 54 08" src="https://user-images.githubusercontent.com/101993818/194139622-8ce93f7e-e1d5-4ec8-9b48-8e442b7beba9.png">

**Comment by jrafaaael** [comment]
I'm able to work on that feature!

**Comment by clebsonf** [comment]
@Gw3i 
This idea is fantastic, that's where we want to go! Could you take the lead or create more requirements or implement them?

**Comment by Gw3i** [comment]
@clebsonf Yes, for sure. I'll write down the requirements 👍 I would like to write a part of the code along with others who want to 😊 Could you add the #hacktoberfest label? Thanks in advance!

**Comment by Gw3i** [comment]
Hey @jrafaaael, 
do you want to implement the whole feature or split it, so we are not coding the same at once? 😄

**Comment by jrafaaael** [comment]
@Gw3i I can work on the first two requirements listed and you in the next two. Do you agree?

**Comment by Gw3i** [comment]
@jrafaaael Sounds good 👍

**Comment by clebsonf** [comment]
@Gw3i I'll send you an invite as a maintaining member, what do you think?

**Comment by Gw3i** [comment]
@clebsonf Sounds good 👍

**Comment by clebsonf** [comment]
> @clebsonf Sounds good 👍

I sent the invite, could you check and accept it?
